src/main/resources/AdminDashboard.fxml

```diff
@@ -0,0 +1,100 @@
+                            <TableColumn prefWidth="127.0" text="Username" />
+                            <TableColumn minWidth="0.0" prefWidth="161.0" text="Employee ID" />
+                            <TableColumn prefWidth="84.0" text="Date/Time" />
+                        </columns>
+                    </TableView>
+                </children>
+            </VBox>
+         </children>
+      </AnchorPane>
+   </center>
+   <top>
+      <ToolBar prefHeight="127.0" prefWidth="1136.0" BorderPane.alignment="CENTER">
+         <items>
+            <ImageView fitHeight="131.0" fitWidth="99.0" pickOnBounds="true" preserveRatio="true">
+               <image>
+                  <Image url="@logo.png" />
+               </image>
+            </ImageView>
+            <Button mnemonicParsing="false" text="Patient&#10;Registration">
+               <font>
+                  <Font size="14.0" />
+               </font>
+            </Button>
+            <Button mnemonicParsing="false" text="Patient&#10;Lookup">
+               <font>
+                  <Font size="14.0" />
+               </font>
+            </Button>
+            <Button mnemonicParsing="false" text="Doctor&#10;Lookup">
+               <font>
+                  <Font size="14.0" />
+               </font>
+            </Button>
+            <Button mnemonicParsing="false" prefHeight="50.0" text="User Lookup">
+               <font>
+                  <Font size="14.0" />
+               </font>
+            </Button>
+            <Button mnemonicParsing="false" text="Register&#10;New User">
+               <font>
+                  <Font size="14.0" />
+               </font>
+            </Button>
+            <Button mnemonicParsing="false" text="Lab&#10;Requests">
+               <font>
+                  <Font size="14.0" />
+               </font>
+            </Button>
+            <Button mnemonicParsing="false" prefHeight="50.0" prefWidth="70.0" text="Logs">
+               <font>
+                  <Font size="14.0" />
+               </font>
+            </Button>
+            <Pane prefHeight="131.0" prefWidth="678.0" />
+            <VBox alignment="TOP_CENTER" prefHeight="0.0" prefWidth="93.0" spacing="5.0">
+               <children>
+                  <ImageView fx:id="userPicture" fitHeight="83.0" fitWidth="100.0" onMouseClicked="#onProfilePictureClicked" pickOnBounds="true" preserveRatio="true">
+                     <image>
+                        <Image url="@deafaultProfilePicture.png" />
+                     </image>
+                     <VBox.margin>
+                        <Insets top="10.0" />
+                     </VBox.margin>
+                  </ImageView>
+                  <MenuButton fx:id="userOptions" mnemonicParsing="false" text="Options">
+                     <items>
+                        <MenuItem mnemonicParsing="false" text="Profile" />
+                        <MenuItem mnemonicParsing="false" text="Log Out" />
```

Comment:
we have log-out but sign in, should we make the login page submit buttons "Log In".